javascript/node/selenium-webdriver/BUILD.bazel

```diff
@@ -88,3 +89,41 @@ genrule(
     ],
     cmd = "cp $(locations //:license) $(@D)",
 )
+
+nodejs_binary(
+    name = "cdp-srcs-generator",
+    entry_point = "devtools/generator/protocol-dts-generator.js",
+    data = [
+       ":browser_protocol",
+       ":js_protocol"
+    ],
+)
+
+copy_file(
+    name = "browser_protocol",
+    src = "//common/devtools:browser_protocol.json",
+    out = "devtools/generator/browser_protocol.json",
+)
+
+copy_file(
+    name = "js_protocol",
+    src = "//common/devtools:js_protocol.json",
+    out = "devtools/generator/js_protocol.json",
+)
+
+genrule(
+    name="create-cdp-srcs",
```

Comment:
this rule needs to be called by `selenium-webdriver` and the `jasmine_node_test` rule